src/index.html

```diff
@@ -1,16 +1,46 @@
-<!doctype html>
+<!Doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body> 
+    <form 
+      action="#" 
+      method="get" 
+      data-qa="big" 
+      class="search">
+      <label>
```

Comment:
```suggestion
    <form 
      action="#" 
      method="get" 
      data-qa="big" 
      class="search"
    >
      <label>
```
<img width="1044" alt="image" src="https://user-images.githubusercontent.com/102413088/184665018-ea3f8fc9-2c56-492f-b8c6-83abc194a8c0.png">

будь-ласка, перевір форматування html згідно чекліста пункт 5
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md